src/transaction/declare.rs

```diff
@@ -222,24 +225,28 @@ impl Declare {
             0,
         );
 
-        let call_info = entry_point.execute(
+        let ExecutionResult { call_info, .. } = entry_point.execute(
             state,
             block_context,
             resources_manager,
             &mut self.get_execution_context(block_context.invoke_tx_max_n_steps),
-            false,
+            true,
+            block_context.validate_max_n_steps,
         )?;
 
+        // TODO: handle reverted transactions
```

Comment:
I don't understand this. Wasn't this the goal of this PR?

Reply:
My bad, the support_reverted should have been false in this case since we want to revert the execution and not the validation